docs/audits.md

```diff
@@ -642,6 +642,31 @@ In general, users should use for [Github Actions environment files]
         echo "$HOME/.local/my-bin" >> "$GITHUB_PATH"
     ```
 
+## `github-env`
+
+| Type     | Examples           | Introduced in | Works offline  | Enabled by default |
+|----------|--------------------|---------------|----------------|--------------------|
+| Workflow  | [github-env.yml]   | v0.6.0        | ✅             | ✅                 |
+
+[github-env.yml]: https://github.com/woodruffw/gha-hazmat/blob/main/.github/workflows/github-env.yml
+
+Detects dangerous usages of `$GITHUB_ENV` environment variable.
+
+This audit compounds over `dangerous-triggers`: given the proper circumstances, an attacker
+may achieve code execution by controlling this environment variable.
```

Comment:
```suggestion
When used in workflows with dangerous triggers (such as `pull_request_target` and `workflow_run`), `GITHUB_ENV` can be an arbitrary code execution risk.

In particular, if the attacker is able to set arbitrary variables or variable contents via `GITHUB_ENV`, they made be able to set `LD_PRELOAD` or otherwise induce code execution implicitly within subsequent steps.
```